### PR TITLE
Organisation Form

### PR DESCRIPTION
**Pull request by sumitFluxbyte**
create a form basic form and validation with basic responsive design

### CODE REVIEW

File: packages/frontend/src/pages/organisation/index.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import React, { useState } from "react";
+import organisationImage from "../../assets/png/organisation.png";
+import backGroundImage from "../../assets/png/background2.png";
+import OrganisationForm from "./organisationForm";
+function Organisation() {
+  const [organisationForm, setOrganisationForm,] = useState<boolean>(false);
+  return (
+    <div>
+      <div
+        className={`w-full h-screen flex justify-center items-center bg-[url(/src/assets/png/background2.png)] bg-cover bg-no-repeat`}
```
Comment (harsh-fluxbyte) [inline packages/frontend/src/pages/organisation/index.tsx:10]: bg-[url(/src/assets/png/background2.png)] a working che?

File: packages/frontend/src/pages/organisation/index.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        style={{ backgroundImage: backGroundImage, }}
+      >
+        <div className="flex flex-col justify-center items-center gap-24">
+          <img
+            src={organisationImage}
+            alt={organisationImage}
```
Comment (harsh-fluxbyte) [inline packages/frontend/src/pages/organisation/index.tsx:16]: alt string hoi

File: packages/frontend/src/pages/organisation/index.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+          <div className="lg:flex justify-between gap-x-5">
+            <div>
+              <span className="text-primary-400 text-3xl font-medium font-['Poppins']">
+                Don’t have any organisation?
+                <br />
+              </span>
+              <span className="text-neutral-600 text-5xl font-bold">
+                Create One!
+              </span>
+              <span className="text-neutral-500 text-6xl font-semibold"> </span>
```
Comment (harsh-fluxbyte) [inline packages/frontend/src/pages/organisation/index.tsx:28]: A span nu su kam che?

File: packages/frontend/src/pages/organisation/organisationForm/index.tsx [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+    onSubmit: (values) => {},
+  });
+  return (
+    <div className="absolute w-full h-full top-full left-full -translate-x-full -translate-y-full flex justify-center items-center bg-primary-900 bg-opacity-50 ">
+      <div className="bg-white rounded-lg shadow-md px-2.5 md:px-6 lg:px-8 pt-6 pb-8 mb-4 md:w-3/4 w-11/12 lg:w-[40rem]">
+        <div className="flex justify-between my-1 mb-5">
+          <h1 className="text-2xl lg:text-3xl font-bold text-gray-500">
+            Create new organisation
+          </h1>
+          <button onClick={props.close}>
```
Comment (harsh-fluxbyte) [inline packages/frontend/src/pages/organisation/organisationForm/index.tsx:37]: props.close na use kar. Upar destructure kari ne use kar

**Review by harsh-fluxbyte — CHANGES_REQUESTED** [review]

File: packages/frontend/src/pages/organisation/index.tsx (hunk 1 above)
Comment (sumitFluxbyte) [reply packages/frontend/src/pages/organisation/index.tsx:10]: yes, it is working

Comment (harsh-fluxbyte) [reply packages/frontend/src/pages/organisation/index.tsx:10]: Nai chaltu hoi remove kari jo e tag to pan background avse.

File: packages/frontend/src/pages/organisation/index.tsx (hunk 3 above)
Comment (sumitFluxbyte) [reply packages/frontend/src/pages/organisation/index.tsx]: for text copy code for figma it is add by accidentally

**Review by harsh-fluxbyte — APPROVED** [review]

**Pull request merged**